changelog/@unreleased/pr-6414.v2.yml

```
@@ -0,0 +1,5 @@
+type: improvement
+improvement:
+  description: SQLString skips cache lookup when not cachine unregistered queries
```

nit: spelling is hard

```suggestion
  description: SQLString skips cache lookup when not caching unregistered queries
```